public/configs/otherMarketData.json

```diff
@@ -1,4675 +1,24 @@
-    "params": {
-      "id": 9,
-      "ticker": "AVAX-USD",
-      "marketType": "PERPETUAL_MARKET_TYPE_CROSS",
-      "priceExponent": -8.0,
-      "minPriceChange": 2500,
-      "minExchanges": 3,
-      "exchangeConfigJson": [
-        {
-          "exchangeName": "Binance",
-          "ticker": "AVAXUSDT",
-          "adjustByMarket": "USDT-USD"
-        },
-        {
-          "exchangeName": "Bitstamp",
-          "ticker": "AVAX/USD"
-        },
-        {
-          "exchangeName": "Bybit",
-          "ticker": "AVAXUSDT",
-          "adjustByMarket": "USDT-USD"
-        },
-        {
-          "exchangeName": "CoinbasePro",
-          "ticker": "AVAX-USD"
-        },
-        {
-          "exchangeName": "Huobi",
-          "ticker": "avaxusdt",
-          "adjustByMarket": "USDT-USD"
-        },
-        {
-          "exchangeName": "Kraken",
-          "ticker": "AVAXUSD"
-        },
-        {
-          "exchangeName": "Kucoin",
-          "ticker": "AVAX-USDT",
-          "adjustByMarket": "USDT-USD"
-        },
-        {
-          "exchangeName": "Mexc",
-          "ticker": "AVAX_USDT",
-          "adjustByMarket": "USDT-USD"
-        },
-        {
-          "exchangeName": "Okx",
-          "ticker": "AVAX-USDT",
-          "adjustByMarket": "USDT-USD"
-        }
-      ],
-      "liquidityTier": 1,
-      "atomicResolution": -7.0,
-      "quantumConversionExponent": -9,
-      "defaultFundingPpm": 0,
-      "stepBaseQuantums": 1000000,
-      "subticksPerTick": 1000000,
-      "delayBlocks": 3600
-    },
-    "initialDeposit": {
-      "denom": "adydx",
-      "amount": "10000000000000000000000"
-    },
-    "meta": {
-      "assetName": "Avalanche",
-      "referencePrice": 34.30079483175494
-    }
-  },
-  "TRX": {
-    "title": "Add TRX-USD perpetual market",
-    "summary": "Add the x/prices, x/perpetuals and x/clob parameters needed for a TRX-USD perpetual market. Create the market in INITIALIZING status and transition it to ACTIVE status after 3600 blocks. Added via the new market widget.",
-    "params": {
-      "id": 10,
-      "ticker": "TRX-USD",
-      "marketType": "PERPETUAL_MARKET_TYPE_CROSS",
+      "ticker": "MOTHER-USD",
+      "marketType": "PERPETUAL_MARKET_TYPE_ISOLATED",
       "priceExponent": -10.0,
-      "minPriceChange": 2500,
-      "minExchanges": 3,
+      "minPriceChange": 800,
+      "minExchanges": 1,
       "exchangeConfigJson": [
-        {
-          "exchangeName": "Binance",
-          "ticker": "TRXUSDT",
-          "adjustByMarket": "USDT-USD"
-        },
-        {
-          "exchangeName": "Bybit",
-          "ticker": "TRXUSDT",
-          "adjustByMarket": "USDT-USD"
-        },
-        {
-          "exchangeName": "Gate",
-          "ticker": "TRX_USDT",
-          "adjustByMarket": "USDT-USD"
-        },
         {
           "exchangeName": "Huobi",
```

Comment:
just to confirm: we want huobi right? i did see it [here](https://account.huobi.com/price/mother/) but would like to confirm